doc/glossary.rst

```diff
@@ -337,6 +337,10 @@
                 Firefly (v. 0.80). See :ref:`Primary Affinity
                 <rados_ops_primary_affinity>`.
 
+        Quorum	
+                Quorum is the state that obtains when a majority of the
```

Comment:
s/that obtains//

Reply:
I think I'm going to go with "exists", even though I think "that obtains" is, strictly speaking, more correct. I see your point, though. Many readers--especially if their English is weak--probably expect the verb "to obtain" to be transitive.